src/app/drive/services/downloadManager.service.ts

```diff
@@ -409,7 +417,11 @@ export class DownloadManagerService {
         });
         downloadProgress[index] = 1;
 
-        if (res?.allItemsFailed) failedItems.push(driveItem);
+        if (res?.allItemsFailed) {
+          failedItems.push(driveItem);
+        } else if (res?.failedItems.length > 0) {
+          failedItems.push(...(res?.failedItems as DownloadItemType[]));
+        }
```

Comment:
What is the reason for this behaviour? I don't quite understand, possibly I'm missing something